coalahtml/helper.py

```diff
@@ -51,7 +51,7 @@ def create_dir(path: str):
     if not os.path.exists(path):
         try:
             os.mkdir(path, mode=0o775)
-        except OSError as e:
+        except OSError as e:  # pragma: no cover
```

Comment:
If we failed to make directory, does is make sense to continue execution?
If no, don't except or reraise, exception. Let if print the traceback (some may consider this ugly, but IMO the user will understand what happened from it, so it's enough) or handle exception at higher level and print user message.